src/confluent_kafka/src/Admin.c

```diff
@@ -1280,6 +1695,122 @@ Admin_c_ConfigResource_result_to_py (const rd_kafka_ConfigResource_t **c_resourc
         return NULL;
 }
 
+/**
+ * @brief Convert C AclBinding to py
+ */
+static PyObject *
+Admin_c_AclBinding_to_py (const rd_kafka_AclBinding_t *c_acl_binding) {
+        
+        PyObject *args, *kwargs, *AclBinding_type, *acl_binding;
+
+        AclBinding_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                        "AclBinding");
```

Comment:
Error check?

Reply:
Yes. In this case the error happens in the response conversion to python. It should return a KafkaError to be set in the future. The KafkaError has an error code of type _rd_kafka_resp_err_t_ but I'm not sure if one of the internal errors can be used or a new type of error should be added.